src/Command/GuideTestCommand.php

```diff
@@ -55,6 +59,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         foreach ($testClasses as $testClass) {
             $suite->addTestSuite($testClass);
         }
+        if ($executable && !\function_exists('App\Playground\request')) {
+            throw new LogicException(sprintf('The guide "%s" is set to be executable, but no function \'request\' is defined in the \'App\Playground\' namespace', $guide));
```

Comment:
can we not throw, we don't need to add a request if we don't want one, a default `GET /docs` will be sent.

Reply:
Hmm, then this whole PR is actually not needed, should I close it ?

Reply:
I don't the point of getting an executable guide without `request` command

Reply:
I agree with @vincentchalamon, especially considering the Playground.
If the `request` function is not available, then it will also not appear in the Playground and the final user will not be able to easily modify the content of said function for this guide. (Unless he writes the function request AND figures out the good namespace to write it in etc ...)
Imho "executable" implies "Playground", which in turn implies a clearly defined request for the end user